pkg/cvo/updatepayload.go

```diff
@@ -207,6 +207,7 @@ func (r *payloadRetriever) fetchUpdatePayloadToDir(ctx context.Context, dir stri
 					NodeSelector: map[string]string{
 						nodeSelectorKey: "",
 					},
+					PriorityClassName: "openshift-user-critical",
```

Comment:
What is the difference between `system-cluster-critical` vs "openshift-user-critical" ?

Reply:
Never mind, I found https://github.com/openshift/enhancements/blob/master/enhancements/scheduling/priority-class.md

Reply:
After some discussion I think we will be ok with `PriorityClassName: "openshift-user-critical",` as this is an improvement over no priority class. Also we have https://github.com/openshift/cluster-monitoring-operator/blob/6f22c3b33e2af7a61a3d385f0688d270519b49e4/assets/control-plane/prometheus-rule.yaml#L185-L194 and https://github.com/openshift/cluster-monitoring-operator/blob/6f22c3b33e2af7a61a3d385f0688d270519b49e4/assets/control-plane/prometheus-rule.yaml#L25-L40 which will communicate user if the pod is not able to schedule itself.